lib/approval/event_listener.rb

```diff
@@ -30,6 +30,9 @@ def remove_approval_tag(event)
 
     def update_approval_status(event)
       Catalog::NotifyApprovalRequest.new(event.payload['request_id'], event.payload, event.message).process
+    rescue
+      order_item = ApprovalRequest.find_by(:approval_request_ref => event.payload['request_id'])&.order_item
+      order_item&.mark_failed("Internal Error. Check log.")
```

Comment:
There's two things here. First is that we're telling the user to check the log, but I don't see anywhere where we're actually logging the error. I don't even think they have access to logs anyway, right? The second is that I think this message should probably be a bit more user friendly (obviously it's easy to change later, but this doesn't really tell them anything and I can just hear John saying "what does this even mean? This is silly, you might as well have just put 'Problem!'".)

Maybe something like "There was an error while queueing up the next order item #{order_item.name} in the order process."